deployer/scripts/install.sh

```diff
@@ -312,7 +312,7 @@ function generate_es_template(){
     --param ES_RECOVER_EXPECTED_NODES=${es_recover_expected_nodes} \
     --param ES_RECOVER_AFTER_TIME=${es_recover_after_time} \
     --param STORAGE_GROUP_DEFAULT=${storage_group} \
-    $image_params
+    "$image_params"
```

Comment:
Is this code, and the following upgrade.sh and test-upgrade.sh code, even called for 1.5?